pkg/ip/link_linux.go

```diff
@@ -90,7 +90,7 @@ func makeVeth(name, vethPeerName string, mtu int, mac string, hostNS ns.NetNS) (
 			if peerExists(peerName) && vethPeerName == "" {
 				continue
 			}
-			return peerName, veth, fmt.Errorf("container veth name provided (%v) already exists", name)
+			return peerName, veth, fmt.Errorf("container veth name (%v) peer provided (%v) already exists", name, peerName)
```

Comment:
You may need to update the test too.

https://github.com/containernetworking/plugins/blob/670139cffa3075aa42f08c8f1dc2257396205a54/pkg/ip/link_linux_test.go#L154

Reply:
Yep, thanks! This is what I get for opening the PR before tests finished running... :joy: